tests/integration_test/services_test.py

```
@@ -29518,7 +29518,17 @@ def _password_reset(*args, **kwargs):
     )
     actual = response_log.json()
     UserEmailConfirmation(email="integration@demo.ai").save()
-
+    UserActivityLog(
+        type="user_consent",
+        user="integration@demo.ai",
+        message=['Privacy Policy, Terms and Conditions consent'],
+        data={
+            "username": email,
```

Ruff (0.8.2)

Undefined name `email`

(F821)